models/src_linkedin.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 sources:
-  - name: linkedin
+  - name: linkedin # This source will only be used if you are using a single linkedin source connector. If multiple sources are being unioned, their tables will be directly referenced via adapter.get_relation.
```

Comment:
Is this the reason for the fivetran_utils exception update? If so, I would rather we make the update here to be `linkedin_ads`. That is more consistent with what it should be as well. Thoughts?